get-artifacts.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# NB: The latest version of Snowbound is available
+#     [here](https://github.com/ArkCase/arkcase-dependencies/releases). Please
+#     note we modify the vanilla release from Snowbound Software and customize
+#     it for ArkCase's needs. Please contact Bojan Milenkoski for more
+#     information.
+#
+# NB: The version of Tomcat that is used here has been downloaded from [the Apache
+#     archive](https://archive.apache.org/dist/tomcat/tomcat-9/v9.0.50/bin/apache-tomcat-9.0.50.tar.gz).
+
+set -eu -o pipefail
+
+here=$(realpath "$0")
+here=$(dirname "$here")
+cd "$here"
+
+snowbound_version="5.6.2-2021.02.04"
+snowbound_war="VirtualViewerJavaHTML5-${snowbound_version}.war"
+tomcat_version="9.0.50"
+tomcat_major_version="9"
+tomcat_tarball="apache-tomcat-${tomcat_version}.tar.gz"
+
+#XXX
+#jmx_prometheus_agent_version="0.15.0"
+#jmx_prometheus_agent="jmx_prometheus_javaagent-${jmx_prometheus_agent_version}"
+
+#rm -rf artifacts
+#mkdir artifacts
+#
+#echo "Downloading $snowbound_war"
+#aws s3 cp "s3://arkcase-container-artifacts/ark_snowbound/artifacts/${snowbound_war}" artifacts/
+
+echo "Downloading $tomcat_tarball"
+aws s3 cp "s3://arkcase-container-artifacts/ark_snowbound/artifacts/${tomcat_tarball}" artifacts/
```

Comment:
For now we can move forward with this, but the hope is to do away with this script. Specifically: Tomcat should be readily available at all times due to its huge popularity... so let's just pull it from the Dockerfile, from the source. We can use ARG values to help parameterize things as well.

Reply:
@JonHolman Have we made a decision on the issue of downloading artifacts from the original website vs our S3 bucket?

Reply:
I thought we had - avoid S3 where possible/feasible, use S3 in public access mode (direct link, no browsing) when not ...? Did I misunderstand that?

Reply:
I think so, like Diego said.  If its already publicly available, use that, if its not put it in our bucket, but we still need to make our bucket public.  Feel free to do that if its blocking you.  We can also go ahead and empty our bucket and start over.

Reply:
Starting over is probably a good idea - will help us clean out cruft in one pass, and the work required to add the missing artifacts is trivial (just re-upload the required file).

Reply:
I remove all the artifacts stuff. I am now downloading Snowbound directly from github and Tomcat from archive.apache.org. I also added the `fonts.tar.gz` file to this repo (it's only 11MiB in size).

Reply:
I agree that an 11MB binary isn't too bad a thing to include in the repo. Cool!